webapp/channels/src/components/user_group_popover/group_member_list/group_member_list.tsx

```diff
@@ -171,13 +179,20 @@ const GroupMemberList = (props: Props) => {
                         hideStatus={user.is_bot}
                     >
                         <UserButton>
-                            <Avatar
-                                username={user.username}
-                                size={'sm'}
-                                url={Utils.imageURLForUser(user?.id ?? '')}
-                                className={'avatar-post-preview'}
-                                tabIndex={-1}
-                            />
+                            <span className='status-wrapper'>
+                                <Avatar
+                                    username={user.username}
+                                    size={'sm'}
+                                    url={Utils.imageURLForUser(user?.id ?? '')}
+                                    className={'avatar-post-preview'}
+                                    tabIndex={-1}
+                                />
+                                <StatusIcon
+                                    className='status user-popover-status'
+                                    status={status}
+                                    button={true}
```

Comment:
The `button` property seems to only remove the `status` class, and we are adding it again in the class name. Therefore... shouldn't it work the same way omitting the `button` prop and the `status` part of the classname?